packages/legend-query-builder/src/components/__tests__/QueryBuilderFetchStructure.test.tsx

```diff
@@ -1309,74 +1307,6 @@ test(
   },
 );
 
-test(
-  integrationTest(
-    'Query builder filter supports actions/flows for is in clause',
```

Comment:
I removed this test as it's already covered by tests in `packages/legend-query-builder/src/components/__tests__/QueryBuilderFilterPanel.test.tsx`.